docs/manifest.json

```diff
@@ -2,1297 +2,1297 @@
 	{
 		"title": "Gutenberg Handbook",
 		"slug": "handbook",
-		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/readme.md",
+		"markdown_source": "../docs/readme.md",
```

Comment:
Thinking if this should be just `docs/readme.md` (relative to the root of the repository instead of being relative to the file). I wonder which one is simpler for the importer. cc @dd32

Reply:
The docs importer only accepts an absolute URI here. https://meta.trac.wordpress.org/browser/sites/trunk/wordpress.org/public_html/wp-content/themes/pub/gutenberg/inc/docs-importer.php#L25